### PR TITLE
App badges in details screen

### PR DESCRIPTION
**Pull request by delfipolito**
Put TargetAppBage in a separate component so it can be used in the voote cards and in the details screen

image

### CODE REVIEW

File: src/components/Proposals/ProposalDetails/ProposalDetails.js [hunk 1]
```diff
@@ -79,15 +80,17 @@ function ProposalDetails({ vote }) {
...
+                display: flex;
+                justify-content: space-between;
+              `}
+            >
+              <TargetAppBadge vote={vote} />
+              {youVoted && (
                 <Tag icon={<IconCheck size="small" />} label="Voted" />
-              </div>
-            )}
+              )}
```
Comment (andy-hook) [inline src/components/Proposals/ProposalDetails/ProposalDetails.js:92]: Just checking, are we keeping the "You voted" tag? I might be wrong but I thought this app badge might be replacing it.

File: src/components/Proposals/TargetAppBadge.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { AppBadge, GU } from '@aragon/ui'
+import { useDescribeVote } from '../../hooks/useDescribeVote'
+import { useOrgApps } from '../../providers/OrgApps'
+import { getAppPresentation } from '../../utils/app-utils'
+import LoadingSkeleton from '../Loading/LoadingSkeleton'
+
+function TargetApp({ vote }) {
```
Comment (andy-hook) [inline src/components/Proposals/TargetAppBadge.js:9]: Let's give this component the same name `TargetAppBadge`

File: src/components/Proposals/TargetAppBadge.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  const {
+    emptyScript,
+    targetApp,
+    loading: descriptionLoading,
```
Comment (andy-hook) [inline src/components/Proposals/TargetAppBadge.js:13]: We can probably just go with `loading` in this new context, it's fairly obvious now what the loading refers to :)

File: src/components/Proposals/TargetAppBadge.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+  } = useDescribeVote(vote.script, vote.id)
```
Comment (andy-hook) [inline src/components/Proposals/TargetAppBadge.js:14]: We should generally try to only pass the props needed by a component, I think here is a good chance to pass just `voteScript` and `voteId` as props.

**Review by andy-hook — APPROVED** [review]
Couple of small things we can improve now that it has its own component but otherwise LGTM!

File: src/components/Proposals/ProposalDetails/ProposalDetails.js (hunk 1 above)
Comment (delfipolito) [reply src/components/Proposals/ProposalDetails/ProposalDetails.js:92]: Hey, no. They both should be there. One is the target app and the other is the tag that appears when you already voted in the vote. :) like this voting app: 

image

**Pull request merged**